test/integ/test_cursor.py

```diff
@@ -1259,6 +1259,24 @@ def test_resultbatch(
                     t.message["type"] == TelemetryField.GET_PARTITIONS_USED
                     for t in telemetry_data.records
                 )
+        telemetry_data = []
+        add_log_mock = mock.Mock()
+        add_log_mock.side_effect = lambda datum: telemetry_data.append(datum)
+        con._telemetry.add_log_to_batch = add_log_mock
```

Comment:
I believe that this is a mistake here, as it does what the code above does, without the usage of `capture_sf_telemetry`

Reply:
You're right, sorry. Removed this old code and kept the version that uses `capture_sf_telemetry`. Both versions got mixed up with the rebase